src/main/java/com/alexx/employeesAndDepartments/controllers/advices/ValidationErrorControllerAdvice.java

```diff
@@ -26,6 +28,7 @@ public ValidationErrorResponse onConstraintValidationException(ConstraintViolati
                         violation.getMessage()
                 )
         ).collect(Collectors.toList());
+        log.warn(violations.toString());
```

Comment:
Please read again logging best practices. This time please look at examples. At many examples

Reply:
Please, give me a link with best examples of logging in Rest api